demo/settings_DE.yaml

```diff
@@ -0,0 +1,67 @@
+# This is an example of a feature-complete JobFunnel configuration YAML.
+# Try this out by simply running: "funnel load -s demo/settings.yaml"
+
+# Path where your master CSV, block-lists, and cache data will be stored
+# NOTE: we create any missing directories in these filepaths
+master_csv_file: demo_job_search_results/demo_search.csv
+cache_folder: demo_job_search_results/cache
+block_list_file: demo_job_search_results/demo_block_list.json
+duplicates_list_file: demo_job_search_results/demo_duplicates_list.json
+log_file: demo_job_search_results/log.log
+
+# Job search configuration
+search:
+
+  # Locale settings, one of USA_ENGLISH, CANADA_ENGLISH, CANADA_FRENCH:
+  # This tells JobFunnel where the website we are scraping is located, and
+  # what language the contents are in.
+  locale: GERMANY_GERMAN
+
+  # Job providers which we will search, one of INDEED, MONSTER, GLASSDOOR:
+  # NOTE: we choose domain via locale (i.e. CANADA_ENGLISH -> www.indeed.ca)
+  # FIXME: we need to add back GLASSDOOR when that's working again.
+  providers:
+    - INDEED
+
+  # Region that we are searching for jobs within:
+  province_or_state: "108"  # NOTE: this is generally 2 characters long.
+  city: "Berlin"  # NOTE: this is the full city / town name.
+  radius: 0  # km (NOTE: if we were in locale: USA_ENGLISH it's in miles)
+
+  # These are the terms you would be typing into the website's search field:
+  keywords:
+    - community-manager
```

Comment:
Maybe we can change this to match the other pre-configured configs by searching for just Python? This helps with CI as we are almost always guaranteed to have results.

Reply:
I changed this to the same keywords of the US indeed scraper. There are around 100 results and around 3 pages, thereby testing crawling of pagination while not crawling for a longer period.